plugins/json/src/main/java/org/apache/struts2/json/JSONResult.java

```diff
@@ -68,8 +69,6 @@
  */
 public class JSONResult implements Result {
 
-    private static final long serialVersionUID = 8624350183189931165L;
```

Comment:
Since `JSONResult` still extends `Serializable`, maybe the `serialVersionUID` should be kept ?

By removing it, anyone who Java-serialized instances in 2.5.x (possible) and then tries to Java-deserialize them in 2.6 could experience deserialization failures.

If this change is intentional because of the logic change for 2.6, maybe a brand new `serialVersionUID` value should be generated instead of just dropping it ?

Reply:
Yeah, the idea was as this is a larger change to the class the ID should be changed, then I thought _why do we need the ID at all here, nobody should ever serialize a instance of `Result`_. So probably the main case is to stop extending `Serializable` by `Result`. I will put back a new ID.